packages/www/pages/docs/guides/debugging-guide.mdx

```diff
@@ -0,0 +1,71 @@
+import DocsLayout from "../../../components/DocsLayout";
+
+<DocsLayout>
+
+
+# Debugging Livepeer.com Stream Playback
+
+This step-by-step guide will help you debug common stream playback problems. Try these steps before reaching out to Livepeer.com directly.
+
+We will use the [Livepeer.com test player](/app/test-player) to debug. You must be logged into the Livepeer.com dashboard. 
+
+#### Prerequisite: Locate your playback URL. 
+
+Your playback URL is specific to the stream ingest region. 
+- This [Livepeer.com API guide](/docs/guides/api/playback-a-live-stream) touches on finding your playback URL by combining a base url with a `playbackId`. If you don't know your stream `playbackId`, you can retrieve it via [`GET /stream/<id>`](/docs/api/stream/get-stream).
+- You can also find the playback URL on the Livepeer.com dashboard stream page, `https://livepeer.com/app/stream/<stream-id>`. Learn more [here](/docs/guides/dashboard/playback-a-stream).
```

Comment:
We could add a third bullet point for the Livepeer beginners, something like this: "- For a complete list of your streams, go to your Livepeer dashboard at https://livepeer.com/app/user. Click on the stream you're debugging and find your playback URL there."

Reply:
Agreed, but I don't want to add that until we have pagination. For someone who creates many streams, this solutions won't work. Pagination will hopefully be part of the dashboard and admin tweaks that happen end of december/early january. Making a note in my records to add this bullet point once we have that feature in the streams list.

Reply:
That makes sense, good to hear on pagination